src/libs/actions/User.ts

```diff
@@ -138,55 +149,51 @@ function requestContactMethodValidateCode(contactMethod) {
         },
     ];
 
-    API.write(
-        'RequestContactMethodValidateCode',
-        {
-            email: contactMethod,
-        },
-        {optimisticData, successData, failureData},
-    );
+    type RequestContactMethodValidateCodeParams = {email: string};
+
+    const parameters: RequestContactMethodValidateCodeParams = {email: contactMethod};
+
+    API.write('RequestContactMethodValidateCode', parameters, {optimisticData, successData, failureData});
 }
 
 /**
- * Sets whether or not the user is subscribed to Expensify news
- *
- * @param {Boolean} isSubscribed
+ * Sets whether the user is subscribed to Expensify news
  */
-function updateNewsletterSubscription(isSubscribed) {
-    API.write(
-        'UpdateNewsletterSubscription',
+function updateNewsletterSubscription(isSubscribed: boolean) {
+    type UpdateNewsletterSubscriptionParams = {isSubscribed: boolean};
+
+    const parameters: UpdateNewsletterSubscriptionParams = {isSubscribed};
```

Comment:
NAB: We usually put params closers API.write/API.read, under optimisticData, failureData and successData